core/src/main/scala/kafka/log/LogManager.scala

```diff
@@ -379,7 +391,7 @@ class LogManager(logDirs: Seq[File],
       threadPools.foreach(_.shutdown())
     }
 
-    info(s"Logs loading complete in ${time.milliseconds - startMs} ms.")
+    info(s"Log loading completed after ${time.milliseconds - startMs}ms.")
```

Comment:
Could we maybe say something like `Loaded n logs after...` where `n` is the number of logs we loaded?